GOVERNANCE.md

```diff
@@ -0,0 +1,235 @@
+- [Users](#users)
+- [Contributors](#contributors)
+- [Triagers](#triagers)
+- [Code Owners](#code-owners)
+
+### Users
+
+Users are expected to follow the [Code of Conduct](CODE_OF_CONDUCT.md) and are encouraged to participate in the
+community.
+
+Users are community members who require the operational functionality of Llama Stack. They are the most important
+community members, and without them, the project would have no purpose. Anyone can be a user; there are no special
+requirements.
+
+Llama Stack asks its users to participate in the project and community as much as possible. User contributions enable
+the project team to ensure that they satisfy the needs of those users. Frequently, user contributions include (but are
+not limited to):
+* Providing developers with feedback on the project (i.e., the user experience)
+* Providing feature requests
+* Filing bug reports or flagging issues
+* Providing moral support
+* Evangelizing the project (e.g., On Social Media)
+
+Users who continue to engage with the project and its community will often become more and more involved. Such users may
+find themselves becoming contributors, as described in the next section.
+
+### Contributors
+
+Contributors are community members who contribute in concrete ways to the project. Anyone can become a contributor, and
+contributions can take many forms, as detailed in the [all-contributors project](https://allcontributors.org/docs/en/emoji-key#table).
+There is no expectation of commitment to the project, no specific skill requirements, and no selection process.
+
+In addition to their actions as users, contributors may also find themselves doing one or more of the following:
+*   Supporting new users (existing users are often the best people to help new users)
+*   Creating, triaging or commenting on GitHub Issues
+*   Doing code reviews or commenting on technical documents
+*   Writing, editing, translating or reviewing the documentation
+*   Organizing events or evangelizing the project (e.g,. on Social Media)
+
+Contributors engage with the project through the issue tracker or by writing or editing documentation. They submit changes to the project itself via Pull Requests (PRs), which will be considered for inclusion in the project by existing Code Owners (see next section).
+
+Contributors should follow the following guides when creating PRs:
+- [Contribution Process](CONTRIBUTING.md)
+
+As contributors gain experience and familiarity with the project, their profile and commitment within the community will
+increase. At some stage, they may find themselves being nominated to become a Triager or Code Owner by an
+existing Code Owner.
+
+### Triagers
+Triagers are contributors who have shown that they are committed to Llama Stack's continued development by helping to
+manage the project's backlog. They are responsible for ensuring that the project's issues are well-organized and up to
```

Comment:
might be good to clarify what "manage" means

Reply:
Added:

```
(track, label, assign issues - general issue hygiene) 
```